img_up/models.py

```diff
@@ -0,0 +1,19 @@
+from django.db import models
+from uuid import uuid4
+from django.contrib.auth.models import User
+
+
+# Create your models here.
+class Epilepsy(models.Model):
+    id = models.UUIDField(primary_key=True, default=uuid4, editable=False)
+    admin_name = models.CharField(max_length=30)
+    state = models.CharField(max_length=30)
+    numOfTotalCases = models.IntegerField()
```

Comment:
I really like the resource you have chosen to model for your API. In this case, `numOfTotalCases` would be the sum of adult cases and child cases, right? 

Conceptually, you'd just have the number of adult and child cases, and if you ever needed the total, you'd simply run a query that would sum both of them. This is because if you save a property which is derived from two other properties, you might have an anomaly where either the adult or child number changes, but the total might stay the same, or vice versa.

But there's also tradeoffs: if you know you're going to access this property _a lot_, it may make sense to just have a total after all. I think in this case, adding two numbers won't be a big deal, so I'd feel better with just a query getting this data. But it's worth thinking about this tension between performance and integrity.